software/tracksight/frontend/src/app/page.tsx

```diff
@@ -2,21 +2,36 @@
 
 import { useEffect, useState, React } from 'react';
 import { io } from "socket.io-client";
-import { Layout } from 'antd';
+import { Layout, Divider, Button } from 'antd';
 const { Header, Content } = Layout;
 
 import styles from './page.module.css';
 import NavBar from './components/navbar.tsx';
 import Graph from './components/graph.tsx';
+import Dashboard from './components/dashboard.tsx';
 import DropdownMenu from './components/dropdown_menu';
 
 const Home = () => {
     const [componentToDisplay, setComponentToDisplay] = useState("visualize");
     const [socketInstance, setSocketInstance] = useState("");
     const [loading, setLoading] = useState(true);
+    const [graphs, setGraphs] = useState([]);
+
+    // logic to add a new graph
+    const addGraph = () => {
+        const newGraphId = Date.now();  
+        setGraphs(prevGraphs => [...prevGraphs, newGraphId]);
+    };
+
+    const deleteGraph = (graphId) => {
+        setGraphs(prevGraphs => prevGraphs.filter(id => id !== graphId));
+    };
+    
 
     useEffect(() => {
-        const socket = io("localhost:5000/", {
+        // NOTE -> io address may have to change depending on where your server is run. Once you run the server, it will tell you where it is running
+        // i.e Running on http://127.0.0.1:5000, so you would change the io address to that
+        const socket = io("http://127.0.0.1:5000/", {
```

Comment:
maybe there is a way to automate this, but can be a later PR or something to investigate later

Reply:
Oh good catch, I think this should be changed back to `localhost`. Did that not work for you for some reason? I believe the idea behind `localhost` is to solve the changing ip address problem. It always references what it is on your computer